test/examples/make.jl

```diff
@@ -395,3 +395,22 @@ else
     @info "Skipping build: LaTeXWriter/latex_texonly" EXAMPLE_BUILDS get(ENV, "DOCUMENTER_TEST_EXAMPLES", nothing)
     nothing
 end
+
+examples_latex_headings_doc = if "latex_headings" in EXAMPLE_BUILDS
+    @info("Building mock package docs: LaTeXWriter/latex_headings")
+    @quietly makedocs(
+        format = Documenter.Writers.LaTeXWriter.LaTeX(platform = "none"),
+        sitename = "Documenter LaTeX Heading Tests",
+        root  = examples_root,
+        build = "builds/latex_headings",
```

Comment:
It would be nice to move the heading `.md` files into a separate source directory (e.g. `src.latex_headings) and then put a `source = "src.latex_headings"` here (like we have for `latex_simple`).